sys-fs/mtd-utils/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST mtd-utils-1.5.2.tar.gz 343843 BLAKE2B 7f6612c4b9185583e9bbc785f7f96b08b7897c4af0c442a6206a446dc25f3c2aae9670c9344fba09606f73b87c73a96150f77bd974c96abe475d204ba5c28068 SHA512 81eb5cf27814234af65409fe93798ec50d9f733978dc606bd33e63d98eb99afb3f988df47dc66a9368a11baee0a35fa1d0d6677af8c23428919e34bd11a25697
 DIST mtd-utils-2.0.2.tar.gz 355964 BLAKE2B 9a795ddac1287c7c44f52215faa4b6295cf5ac3917392a407c7824f8715f4a180feab95621e6b1aa738fba38950c8e12b722b7c518c3362227e6da25deecd3fb SHA512 e61f4bb24d3b96d39fe262f73e01b7bf3e15a9258fd5766657ef0d807d7b5bb0b489ade9dc45068b2a25bcf378ca2877177df3915398b367f50ff1ef9bc5be7a
+DIST mtd-utils-2.1.2.tar.gz 380370 BLAKE2B 8ad951f5cb6cd853f0cba994c4d14928d88af33f405fabfba814ba6738217f7f50d33da48561ba75be393423281efb4fe934a85eddb951944fa4eb6135e15710 SHA512 e25fb9b1b5cd3eaeda5ca1d122866d54af01830442121229c8d40d36fc3eeb2dc46f6e0dff7790af115315c7027b4062160235f9636cf64164f4c4661a31d337
```

Comment:
```
!!! Fetched file: mtd-utils-2.1.2.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      380333
!!! Expected: 380370
```

Whaat's up with this? Don't tell me it changes everytime it's downloaded based on timestamp or something.

Reply:
Hm, not really sure, I didn't encounter this. I am using regular github tarball, can you please try again now?

Reply:
Clear your distfiles, `rm Manifest` and then use `repoman manifest`. You must have downloaded the one from previous host, because...